stardog-userrole-operator/crd/stardogrole.yaml

```diff
@@ -0,0 +1,123 @@
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    controller-gen.kubebuilder.io/version: v0.3.0
+  creationTimestamp: null
+  name: stardogroles.stardog.vshn.ch
+spec:
+  group: stardog.vshn.ch
+  names:
+    kind: StardogRole
+    listKind: StardogRoleList
+    plural: stardogroles
+    singular: stardogrole
+  scope: Namespaced
+  subresources:
+    status: {}
+  validation:
+    openAPIV3Schema:
+      description: StardogRole is the Schema for the stardogroles API
+      properties:
+        apiVersion:
+          description: 'APIVersion defines the versioned schema of this representation
+            of an object. Servers should convert recognized schemas to the latest
+            internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#resources'
+          type: string
+        kind:
+          description: 'Kind is a string value representing the REST resource this
+            object represents. Servers may infer this from the endpoint the client
+            submits requests to. Cannot be updated. In CamelCase. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#types-kinds'
+          type: string
+        metadata:
+          type: object
+        spec:
+          description: StardogRoleSpec defines the desired state of StardogRole
+          properties:
+            permissions:
+              description: Permissions lists the permissions assigned to a role
+              items:
+                description: StardogPermissionSpec defines a Stardog permission assigned
+                  to a Role
+                properties:
+                  action:
+                    description: Action describes the action a specific permission
+                      is assigned to
+                    enum:
+                      - ALL
+                      - CREATE
+                      - DELETE
+                      - READ
+                      - WRITE
+                      - GRANT
+                      - REVOKE
+                      - EXECUTE
+                    type: string
+                  resourceType:
+                    description: ResourceType describes the type of resource a specific
+                      permission is assigned to
+                    enum:
+                      - DB
+                      - USER
+                      - ROLE
+                      - ADMIN
+                      - METADATA
+                      - NAMEDGRAPH
+                      - VIRTUALGRAPH
+                      - ICVCONSTRAINTS
```

Comment:
I think we can skip this feature for now. Especially considering the fact that it would require the software itself to be updated as well not just the helm chart.